cmd/notify-webhook.go

```diff
@@ -91,10 +91,10 @@ func isNetErrorIgnored(err error) bool {
 }
 
 // Lookup endpoint address by successfully POSTting
-// a JSON which would send out minio release.
+// empty body.
 func lookupEndpoint(urlStr string) error {
-	minioRelease := bytes.NewReader([]byte(ReleaseTag))
-	req, err := http.NewRequest("POST", urlStr, minioRelease)
+	emptyBody := bytes.NewReader([]byte(""))
```

Comment:
No need to have a new variable for this just do this 

```
http.NewRequest("POST", urlStr, bytes.NewReader([]byte("")))
```

Reply:
done